source/JsMaterialX/helpers.h

```diff
@@ -13,256 +13,338 @@ std::vector<myClass> arrayToVec(myClass *arr, int size)
 
 // Binding helpers
```

Comment:
The simple `BIND_FUNC_X_Y` macros haven't been changed, just renamed. So most of the changes in this file can be ignored. Only the macros that carry out some work (formerly `BIND_FUNC_X`, now `BIND_X`) have been modified. Furthermore, new macros for static and global functions have been introduced, together with their `_RAW_PTR` version.